src/index/TextIndexReadWrite.h

```diff
@@ -273,7 +271,47 @@ class FrequencyEncode {
 
   // View must be an input range with value type `T`.
   template <typename View>
-  explicit FrequencyEncode(View&& view);
+  requires(!std::same_as<FrequencyEncode, std::remove_cvref_t<View>>)
+  explicit FrequencyEncode(View&& inputView) {
+    auto view = std::forward<View>(inputView);
```

Comment:
That is absolutely a bad idea, it introduces unneeded copies. Please remove this, thecdecov warnings about no forwarding are fine for now.